src/Pages/sangyeonkim/Login/Login.js

```diff
@@ -1,14 +1,113 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import "./Login.scss";
+import { FaFacebookSquare } from "react-icons/fa";
 
 class Loginkim extends Component {
-	state = {  }
-	render() { 
-		return (
-			<div>
-
-			</div>
-		);
-	}
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      pw: "",
+    };
+  }
+
+  goToMain = (e) => {
+    e.preventDefault();
+    if (this.state.id.includes("@") && this.state.pw.length > 4) {
+      fetch("http://10.58.4.128:8000/user/signin", {
+        method: "POST",
+        body: JSON.stringify({
+          email: this.state.id,
+          password: this.state.pw,
+        }),
+      })
+        .then((response) => response.json())
+        .then((result) => console.log("결과: ", result));
+      this.props.history.push("./Main-sangyeonkim");
+    }
+  };
+
+  handleIdInput = (event) => {
+    this.setState({ id: event.target.value });
+  };
+
+  handlePwInput = (event) => {
+    this.setState({ pw: event.target.value });
+  };
+
+  render() {
+    const { id, pw } = this.state;
+    return (
+      <>
+        <section className="container">
+          <section className="structure">
+            <div className="instagram">
+              <p className="westaTitle">Westagrams</p>
+            </div>
+            <div className="loginPage">
+              <div>
+                <input
+                  type="text"
+                  className="id"
+                  placeholder="전화번호, 사용자 이름 또는 이메일"
+                  onChange={this.handleIdInput}
+                />
+              </div>
+              <div>
+                <input
+                  type="password"
+                  className="pw"
+                  placeholder="비밀번호"
+                  onChange={this.handlePwInput}
+                />
+              </div>
+              <div className="login">
+                <button
+                  className={
+                    id.includes("@") && pw.length > 4
+                      ? "loginBtnActive"
+                      : "loginBtn"
+                  }
+                  onClick={(e) => this.goToMain(e)}
```

Comment:
아래와 같이 써도 됩니다. 왜 그럴까요? 생각하시는 이유를 코멘트로 남겨주세요!
```suggestion
                  onClick={this.goToMain}
```

Reply:
지금 보니 제가 왜 넣었는지 이해가 안되네요 ㅠㅠ 아마도 다른 작업을 하다 실수로 넣은 거 같습니다!! detail한 피드백 감사합니다~~